common/src/registers/MachRegister.C

```diff
@@ -864,6 +891,12 @@ namespace Dyninst {
         return is_vec || is_fpr || is_ctl || is_sts;
       }
 
+      case Arch_riscv64: {
+        auto const is_vec = isVector();
+        auto const is_fpr = (category == riscv64::FPR);
+        return is_vec || is_fpr;
```

Comment:
`isFloatingPoint` includes all control/status registers modified or used by FP instructions.

```suggestion
        auto const is_fcsr = (*this == riscv64::fcsr);
        return is_vec || is_fpr || is_fcsr;
```

Reply:
Fixed